WalletWasabi.Gui/Controls/WalletExplorer/CoinListViewModel.cs

```diff
@@ -385,7 +393,20 @@ public CoinListViewModel(Global global, CoinListContainerType coinListContainerT
 					}
 				});
 
-			InitList = ReactiveCommand.Create(() => OnOpen(), outputScheduler: RxApp.MainThreadScheduler);
+			InitList = ReactiveCommand.CreateFromTask(async () =>
+			{
+				try
+				{
+					IsCoinListLoading = true;
+					// We have to wait for the UI to became visible to the user.
+					await Task.Delay(800); // Let other tasks run to display the gui.
```

Comment:
This smells.

Reply:
Yes, it is as before. Approved by @danwalmsley, currently, there is no other solution to this.

Reply:
Not good. Use the pattern as in the `TextResourceViewModelBase`.